articles/meta-keybindings-with-iterm.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Meta keybindings with iTerm
 tags: ["iTerm", "Meta Keys", "Tmux", "Neovim"]
-author: Robert@typecraft.dev
+author: "hey@mthomps4.com"
```

Comment:
Due to ghost and how collaborators must exist for the author, all articles will appear like they are written by me (shown as Typecraft), but the beginning of every article will flag the actual contributor in a byline.

Reply:
Thanks for the explanation.